psql2mysql.py

```diff
@@ -82,12 +81,12 @@ def _query_utf8mb4_rows(self, table, stringColumns):
     def scanTablefor4ByteUtf8Char(self, table):
         stringColumns = self.getStringColumns(table)
         LOG.debug("Scanning Table %s (columns: %s) for problematic UTF8 "
-                 "characters", table.name, stringColumns)
+                  "characters", table.name, stringColumns)
         rows = self._query_utf8mb4_rows(table, stringColumns)
         incompatible = []
         primary_keys = []
-        if not table.primary_key == None:
-          primary_keys = list(table.primary_key)
+        if table.primary_key is not None:
```

Comment:
This is actually fixed in another PR (https://github.com/SUSE/psql2mysql/pull/10)